modules/serverless-functions-using-integrated-registry.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * serverless/serverless-functions-setup.adoc
+
+[id="serverless-functions-using-integrated-registry_{context}"]
+= Building and deploying functions with OpenShift Container Registry
+
+When building and deploying functions, the resulting container image is stored in an image registry. Usually this will be a public registry, such as Quay. However, you can use the integrated OpenShift Container Registry instead if it has been exposed by a cluster administrator.
+
+.Procedure
+
+* Run the `kn func build` or `kn func deploy` commands with the OpenShift Container Registry specified for the `-r` parameter:
++
+.Example build command
+[source,terminal]
+----
+$ kn func build -i <image> -r image-registry.openshift-image-registry.svc
+----
++
+.Example deploy command
+[source,terminal]
+----
+$ kn func deploy -n <namespace> -i <image> -r image-registry.openshift-image-registry.svc
```

Comment:
Similar to above, I don't think you need to include `-n` or `-i` here.
```suggestion
$ kn func deploy -r $(oc get route -n openshift-image-registry)
```

Or, if the env var is set above.

```
kn func deploy -r $REGISTRY
```